bulk_customer_import.py

```diff
@@ -1,23 +1,19 @@
 """
 usage: bulk_customer_import.py [-h] [-l LOGLEVEL]
-                               base_url auth_user auth_pass filename
-                               servicedesk_id
-
+                              base_url auth_user auth_api filename
+                              servicedesk_id
 positional arguments:
   base_url              The url of the hosted instance of JIRA https://yourdomain.atlassian.net
   auth_user             Username for basic http authentication (Account needs to be jira admin)
-  auth_pass             Password for basic http authentication
+  auth_api              API for basic http authentication
```

Comment:
This is for the api token right? Can we call this `auth_token` instead of `auth_api` for clarity?